src/components/List/List.tsx

```diff
@@ -76,11 +80,25 @@ class List extends UIComponent<Extendable<IListProps>, any> {
   // List props that are passed to each individual Item props
   static itemProps = ['debug', 'selection', 'truncateContent', 'truncateHeader', 'variables']
 
+  private focusContainer = ContainerFocusHandler.create(this)
```

Comment:
a bit worrying about the fact that we are using reference on the object that is not fully initialized at this point. While it is fine (as only `props` are used by this method, and those come already initialized), we might still consider to make this call in constructor for better safety.

Reply:
for future refactoring